config/bootstrap/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+# The kubeadm kustomization bundles up all the bits that are kubeadm-specific. 
+# This includes all bootstrap/controlplane components. 
+
+# Adds namespace to all resources.
+namespace: capi-system
+
+bases:
+- crd/
+- rbac/
```

Comment:
Should there be `- webhook/` here as well?

Reply:
surprisingly, there were no webhooks generated for just bootstrap. since there was nothing there, I blew the directory away and removed it from the makefile. can add it back if there's some reason we wanna keep the empty path.